api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters/marxan-directory.service.ts

```diff
@@ -0,0 +1,34 @@
+import { join as joinPath } from 'path';
+import { Injectable } from '@nestjs/common';
+
+import { Workspace } from '../ports/workspace';
+
+import { FileReader } from './file-reader';
+
+@Injectable()
+export class MarxanDirectory {
+  constructor(private readonly reader: FileReader) {}
+
+  get(
+    type: 'INPUTDIR' | 'OUTPUTDIR',
+    within: Workspace,
+  ): {
+    name: string;
+    fullPath: string;
+  } {
+    const config = this.reader.read(
+      joinPath(within.workingDirectory, 'input.dat'),
+    );
+    const matcher = new RegExp(`(${type} )(?<dir>.*)$`, 'gm');
```

Comment:
shouldn't the regex start with `^`?

Reply:
@Dyostiq it could but doesn't matter there (multiline & known format)

Reply:
matters, as the file may contain comments, so:
```
a comment like this suggesting using INPUTDIR something
```
matches the regex

Reply:
Gotcha, you are right!

Reply:
@Dyostiq https://github.com/Vizzuality/marxan-cloud/pull/316/commits/e6738911d900188778e736492f65887266b9682b